### ISSUE TITLE
Numbers drawn with wrong position

### ISSUE THREAD

**Issue by ceduliocezar** [ShawnLin013/NumberPicker#72]
The following image represents the problem. I divided the screen proportionally, and was able to see the that top and bottom numbers are drawn differently.

I've used the code from sample and just add some lines to represent the problem.

![device-2017-08-22-141136](https://user-images.githubusercontent.com/3894020/29578141-98cebdc6-8744-11e7-8aac-013bb58390a9.png)

**Comment by ceduliocezar** [comment]
If you want to reproduce by your self, just change the content of content_main.xml to the following xml code.


```
<?xml version="1.0" encoding="utf-8"?>
<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    android:paddingBottom="@dimen/activity_vertical_margin"
    android:paddingLeft="@dimen/activity_horizontal_margin"
    android:paddingRight="@dimen/activity_vertical_margin"
    android:paddingTop="@dimen/activity_horizontal_margin"
    app:layout_behavior="@string/appbar_scrolling_view_behavior"
    tools:context="com.shawnlin.numberpicker.sample.MainActivity"
    tools:showIn="@layout/activity_main">

    <com.shawnlin.numberpicker.NumberPicker
        android:id="@+id/number_picker"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:layout_centerHorizontal="true"
        android:layout_centerVertical="true" />

    <View
        android:layout_width="match_parent"
        android:layout_height="1dp"
        android:layout_centerInParent="true"
        android:background="#0f0" />

    <View
        android:layout_width="match_parent"
        android:layout_height="1dp"
        android:layout_above="@id/number_picker"
        android:layout_marginBottom="-50dp"
        android:background="#0f0" />

    <View
        android:layout_width="match_parent"
        android:layout_height="1dp"
        android:layout_above="@id/number_picker"
        android:layout_marginBottom="-25dp"
        android:background="#0f0" />

    <View
        android:layout_width="match_parent"
        android:layout_height="1dp"
        android:layout_below="@id/number_picker"
        android:layout_marginTop="-50dp"
        android:background="#0f0" />

    <View
        android:layout_width="match_parent"
        android:layout_height="1dp"
        android:layout_below="@id/number_picker"
        android:layout_marginTop="-25dp"
        android:background="#0f0" />


</RelativeLayout>
```

**Comment by ceduliocezar** [comment]
Seems to me that the problem is related to the propertie selected_text_size.

The bigger the font size the bigger is the difference on top and bottom positioning.

**Comment by ceduliocezar** [comment]
@ShawnLin013 maybe if comparing with the default number picker it becomes more evident.

On the right side of this picture there the default android numberpicker.

![device-2017-08-23-112645](https://user-images.githubusercontent.com/3894020/29621039-01167eb2-87f6-11e7-9830-5fda23a245d7.png)

**Comment by ShawnLin013** [comment]
Fixed at v2.4.6

**Comment by Sherriluneau** [comment]
What if it is not your phone content bitch
